src/App.tsx

```diff
@@ -1,10 +1,61 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
 
-const USER_ID = 0;
+const USER_ID = 10631;
+
+export const filterOptions = ['All', 'Active', 'Completed'];
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState(filterOptions[0]);
+  const [isError, setIsError] = useState(false);
+  const [isHidden, setIsHidden] = useState(false);
+
+  const loadTodos = async () => {
+    setIsError(false);
+    try {
+      const todos = await getTodos(USER_ID);
+
+      setTodosFromServer(todos);
+    } catch {
+      setIsError(true);
+      setTimeout(() => setIsError(false), 3000);
```

Comment:
move to useEffect, check if(isError), after setTimeout and clear timeout in unmount effect